docs/source/using/accessing-the-openshift-docker-registry.adoc

```diff
@@ -0,0 +1,44 @@
+= Accessing the OpenShift Docker Registry
+:icons:
+:toc: macro
+:toc-title:
+:toclevels: 1
+
+OpenShift provides an integrated docker registry which can be used as a local registry.
+
+This helps to speed up the local development as images present in registry can be used for applications running in OpenShift.
+
+== Login to the docker registry 
+
+. Start minishift with the link:../command-ref/minishift_start{outfilesuffix}[`minishift start`] command.
+
+. Add the oc binary to the PATH environment variable.
+
+ $ export PATH=$PATH:~/.minishift/cache/oc/{openshift-version}
+
+. Export the environment variables that are required link:../using/reusing-docker-daemon{outfilesuffix}[to reuse the Docker daemon]
+
+ $ eval $(minishift docker-env)
+
+. Login to the docker registry
+
+ $ docker login -u developer -p `oc whoami -t` 172.30.1.1:5000 
+
+== Staging your application with the OpenShift registry
+
+This scenario is intended to illustrate how to test an application Docker image to a local instance of OpenShift platform that runs in Minishift.
+
+. Build the Docker image as usual.
+
+. Tag the image against the OpenShift registry with the hostname.
+
+ $ docker tag my-app 172.30.1.1:5000/myproject/my-app
+
+. Push the image to the registry to create an image stream with the same name as the application.
+
+ $ docker push 172.30.1.1:5000/myproject/my-app
+
+. Create an application from the image stream and expose the image.
+
+ $ oc new-app --image-stream=my-app --name=my-app
+ $ oc expose service my-app --hostname=my-app.minishift
```

Comment:
Please add a line here summarizing what these steps did or what the user must/can do once he completes these steps. Maybe something like, 'You can now deploy the OpenShift application using the locally built Docker image.